api/lib/controllers/meeting.js

```diff
@@ -265,7 +265,29 @@ module.exports = {
   async index(req, res) {
     const subject_email = req.credentials.user.email;
     const subject_id = req.credentials.sub;
-    const { limit = 0, skip = 0 } = req.query; // Thanks tom
+    const {
+      limit = 0,
+      skip = 0,
+      name = '',
+      owners = []
+    } = req.query;
+
+    const pipelineFilters = [];
+
+    owners.length && pipelineFilters.push({
+      $match:  {
+        owner_id: {
+          $in: owners.map( ( owner ) => ObjectID( owner ) )
+        }
+      }
+    });
+
+    name && pipelineFilters.push({
+      $match: { name: { $regex: name, $options: 'i' } }
+    });
+
+    const sliceStart = parseInt( skip );
+    const sliceEnd = parseInt( limit ) + sliceStart - 1;
```

Comment:
[$slice](https://www.mongodb.com/docs/manual/reference/operator/projection/slice/) doesn't work how it should. It actually works exactly like skip and limit so you can just pass in the `sliceEnd`.